processing/multivariate_comparison.py

```diff
@@ -1191,35 +1259,73 @@ def composite_plotter(
 
         # Plot outliers for 2nd graph
         self.plot(
-            o_ts_ls, o_x_ls, color='r', marker_size=10, marker_fill=False,
-            subplot_pos=(3, 1, 2))
+            o_ts_ls, o_x_ls, color='#9B51E0', marker_size=10,
+            marker_fill=False, subplot_pos=(3, 1, 2))
 
         # Plot third graph
         self.plot(dt_num, y_val, ts_label, y_label, subplot_pos=(3, 1, 3))
 
         # Plot outliers for third graph
         self.plot(
-            o_ts_ls, o_y_ls, color='r', marker_size=10, marker_fill=False,
-            subplot_pos=(3, 1, 3))
+            o_ts_ls, o_y_ls, color='#9B51E0', marker_size=10,
+            marker_fill=False, subplot_pos=(3, 1, 3))
 
         # Set labels and colors for points
         # (label, facecolor, edgecolor)
-        legendInfo = [
-            ('data', '0.75', '0.75'), ('potential outliers', 'None', 'r')]
+        legendInfo = [('data', '0.75', '0.75'),
+                      ('potential outliers', 'None', '#9B51E0')]
 
         # convert legend info to Line Objs
         handles = [mlines.Line2D(
-            [], [], color="None", marker='o', markersize=10, label=l,
+            [], [], color='None', marker='o', markersize=10, label=l,
             markerfacecolor=fc, markeredgecolor=ec)
             for (l, fc, ec) in legendInfo]
         labels = [l for (l, fc, ec) in legendInfo]
 
         # Add labels and colors for point styling
         reg_label = 'Linear Regression between {x} and {y}'.format(
             x=x_label, y=y_label)
-        handles += [mlines.Line2D([], [], color='g', label=reg_label)]
+        handles += [mlines.Line2D([], [], color='#4D91FE', label=reg_label)]
         labels += [reg_label]
-        plt.figlegend(handles, labels, loc='lower center', ncol=3)
+
+        # Calculating analysis details for legend
+        __, ss_total_y = self.compute_sum_of_squares(
```

Comment:
Double underscore here

Reply:
Hi, I'm not quite sure what you mean by this

Reply:
`_` as opposed to `__`

Reply:
So it should be _, ss_total_y (single underscore?)

Reply:
See this for reference : https://stackoverflow.com/questions/11486148/unused-variable-naming-in-python